src/test/java/com/unzer/payment/util/ApplePayAdapterTest.java

```diff
@@ -0,0 +1,15 @@
+package com.unzer.payment.util;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+
+public class ApplePayAdapterTest {
```

Comment:
Is there realy no way to setup a key manager for lokal tests?

Reply:
Currently there is no way to create complex Tests by using mocked Objects. I created a ticket an will do it in a later Step.